internal/util/kubernetes/kubernetes.go

```diff
@@ -274,7 +274,7 @@ func ApplyCRSpec(ctx context.Context, restConfig *rest.Config, object *GenericOb
 			if err != nil {
 				return fmt.Errorf("failed to create Resource %s/%s, err=%v", object.Namespace, object.Name, err)
 			}
-			numaLogger.Debugf("successfully created resource %s/%s", object.Namespace, object.Name)
+			numaLogger.Infof("successfully created resource %s/%s", object.Namespace, object.Name)
```

Comment:
wanted to add at least one line that would be called at Info level when we update something (here for new resource and below for updated resource)